src/main/java/org/opentripplanner/routing/impl/ATLFareServiceImpl.java

```diff
@@ -142,7 +151,6 @@ private float getRidePrice(Ride ride, Fare.FareType fareType, Collection<FareRul
 
     private static class TransferMeta {
         public final int window; // Transfer window in mins
```

Comment:
Similar to the max transfers, the window should be computed using the first ride time of payment.

Reply:
I've done it, but I do feel a little weird about setting all these values on the beginning of the transfer. If MARTA -> XPRESS is a free transfer by some agreement they have, then why should the transfer limit be set by the agency from two trips ago? I think we should look for some clarification here to make sure this is right.

Reply:
Per our discussion the other day, the free transfers are subject to the number of transfers determined by the agency that you paid first (e.g. GCT vs others).